CI/scenarios/volume_scenario.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: kraken-test-pv
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 2Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/data"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kraken-test-pvc
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kraken-test-pod
+spec:
+  volumes:
+    - name: kraken-test-pv
+      persistentVolumeClaim:
+        claimName: kraken-test-pvc
+  containers:
+    - name: kraken-test-container
+      image: 'image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest'
+      volumeMounts:
+        - mountPath: "/home/krake-dir/"
```

Comment:
sorry, one more thing. the directory is missing the end "n" in kraken

Reply:
Merged the PR when this comment was in flight, my bad.

Reply:
No problem!